### PR TITLE
[Docs]Change the code about installation

### PR DESCRIPTION
**Pull request by suibe-qingtian**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -91,10 +91,10 @@ pip install openmim
 mim install mmengine
 mim install "mmcv>=2.0.0rc1"
 mim install "mmdet>=3.0.0rc0"
-# for albumentations
-pip install -r requirements/albu.txt
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# for albumentations
+pip install -r requirements/albu.txt
 mim install -e .
```
Comment (PeterH0323) [inline README_zh-CN.md:98]:
```suggestion
# Install MMYOLO
mim install -e .
```

**Review by PeterH0323 — CHANGES_REQUESTED** [review]

**Review by PeterH0323 — CHANGES_REQUESTED** [review]

**Comment by PeterH0323** [comment]
Hi @suibe-qingtian 
I leave some suggestion for the doc, you can click the `commit suggestion` button if you accept the suggestion change.
Thx for your contribute. 😄

File: README_zh-CN.md (hunk 1 above)
Comment (suibe-qingtian) [reply README_zh-CN.md:99]: Ok，thanks

File: README_zh-CN.md [hunk 2]
```diff
@@ -91,10 +91,10 @@ pip install openmim
 mim install mmengine
 mim install "mmcv>=2.0.0rc1"
 mim install "mmdet>=3.0.0rc0"
-# for albumentations
-pip install -r requirements/albu.txt
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# for albumentations
```
[the remark this answers is not in the document]
Comment (suibe-qingtian) [reply README_zh-CN.md]: ok，thanks

**Comment by PeterH0323** [comment]
Hi @suibe-qingtian 

Pls modifiy the English version at the same time: 

- https://github.com/open-mmlab/mmyolo/blob/main/README.md

And I just find out another wrong code in `get_started.md`, pls do the same change for it :
- https://github.com/open-mmlab/mmyolo/blob/main/docs/en/get_started.md
- https://github.com/open-mmlab/mmyolo/blob/main/docs/zh_cn/get_started.md

image

File: README.md [hunk 3]
```diff
@@ -91,10 +91,10 @@ pip install openmim
 mim install mmengine
 mim install "mmcv>=2.0.0rc1"
 mim install "mmdet>=3.0.0rc0"
-# for albumentations
-pip install -r requirements/albu.txt
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# for albumentations
+pip install -r requirements/albu.txt
```
Comment (PeterH0323) [inline README.md:97]:
```suggestion
# Install albumentations
pip install -r requirements/albu.txt
# Install MMYOLO
```

File: docs/en/get_started.md [hunk 4]
```diff
@@ -61,6 +59,8 @@ Case a: If you develop and run mmdet directly, install it from source:
 ```shell
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# for albumentations
+pip install -r requirements/albu.txt
```
Comment (PeterH0323) [inline docs/en/get_started.md:63]:
```suggestion
# Install albumentations
pip install -r requirements/albu.txt
# Install MMYOLO
```

File: docs/zh_cn/get_started.md [hunk 5]
```diff
@@ -61,6 +59,8 @@ b. 如果使用 albumentations，我们建议使用 pip install -r requirements/
 ```shell
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# for albumentations
+pip install -r requirements/albu.txt
```
Comment (PeterH0323) [inline docs/zh_cn/get_started.md:63]:
```suggestion
# Install albumentations
pip install -r requirements/albu.txt
# Install MMYOLO
```

**Review by PeterH0323 — CHANGES_REQUESTED** [review]

**Review by PeterH0323 — APPROVED** [review]
LGTM

File: docs/zh_cn/get_started.md [hunk 6]
```diff
@@ -61,6 +59,9 @@ b. 如果使用 albumentations，我们建议使用 pip install -r requirements/
 ```shell
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# Install albumentations
+pip install -r requirements/albu.txt
+# Install MMYOLO
 pip install -v -e .
```
Comment (RangeKing) [inline docs/zh_cn/get_started.md:65]:
```suggestion
mim install -v -e .
```
To be consistent with `README.md`

File: docs/en/get_started.md [hunk 7]
```diff
@@ -61,6 +59,9 @@ Case a: If you develop and run mmdet directly, install it from source:
 ```shell
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# Install albumentations
+pip install -r requirements/albu.txt
+# Install MMYOLO
 pip install -v -e .
```
Comment (RangeKing) [inline docs/en/get_started.md:65]:
```suggestion
mim install -v -e .
```
To be consistent with `README.md`

File: README.md [hunk 8]
```diff
@@ -91,10 +91,11 @@ pip install openmim
...
 mim install "mmcv>=2.0.0rc1"
 mim install "mmdet>=3.0.0rc0"
-# for albumentations
-pip install -r requirements/albu.txt
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# Install albumentations
+pip install -r requirements/albu.txt
+# Install MMYOLO
 mim install -e .
```
Comment (RangeKing) [inline README.md:99]:
```suggestion
mim install -v -e .
```

File: README_zh-CN.md [hunk 9]
```diff
@@ -91,10 +91,11 @@ pip install openmim
...
 mim install "mmcv>=2.0.0rc1"
 mim install "mmdet>=3.0.0rc0"
-# for albumentations
-pip install -r requirements/albu.txt
 git clone https://github.com/open-mmlab/mmyolo.git
 cd mmyolo
+# Install albumentations
+pip install -r requirements/albu.txt
+# Install MMYOLO
 mim install -e .
```
Comment (PeterH0323) [inline README_zh-CN.md:99]:
```suggestion
mim install -v -e .
```

**Review by PeterH0323 — CHANGES_REQUESTED** [review]

**Comment by PeterH0323** [comment]
Hi @suibe-qingtian 

Pls change merge branch from `main` to `dev`

**Review by PeterH0323 — APPROVED** [review]
LGTM

**Review by hhaAndroid — APPROVED** [review]

**Comment by OpenMMLab-Assistant005** [comment]
Hi @qingtian5 ！First of all, we want to express our gratitude for your significant PR in the MMYOLO project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**